napari/utils/colormaps/colormap_utils.py

```diff
@@ -374,11 +433,18 @@ def increment_unnamed_colormap(
     -------
     name : str
         Name of colormap after incrementing.
+    display_name : str
+        Display name of colormap after incrementing.
     """
+    display_name = trans._('[unnamed colormap]')
     if name == '[unnamed colormap]':
```

Comment:
does that have any chance to get a translated string in ? in which case this would not work...

Reply:
Nope, name will always be the "english version"